### PR TITLE
Section about upload environments to anaconda.org

### PR DESCRIPTION
**Pull request by malev**
Some instructions about how to use `conda env upload` and `conda env create username/environment`

### CODE REVIEW

File: application/simple/templates/index.html [hunk 1]
```diff
@@ -308,6 +308,39 @@
...
+          <h2>Upload your environment to anaconda.org</h2>
+
+          <p>
+            Now that you have your <code>environment.yml</code> ready, you can upload
+            it to <a href="http://anaconda.org">anaconda.org</a>. You will first need
+            an account there. Go ahead, it's free!
+          </p>
+
+          <p>
+            With your <a href="http://anaconda.org">anaconda.org</a>'s credentials you
```
Comment (pzwang) [inline application/simple/templates/index.html:320]: I think the `'s` is unneeded here.  Also, add a comma after "credentials".

File: application/simple/templates/index.html [hunk 2]
```diff
@@ -308,6 +308,39 @@
...
+            are ready to upload your environment with:
+          </p>
+
+          {% call code.block("bash") %}
+$ conda env upload name-of-the-environment
```
Comment (pzwang) [inline application/simple/templates/index.html:325]: According to the documentation style guide, we should avoid putting the `$` in front of commands.  It sometimes confuses users, and also makes cut & paste harder.

File: application/simple/templates/index.html [hunk 3]
```diff
@@ -308,6 +308,39 @@
...
+... output from conda env upload ...
+          {% endcall %}
+
+          <p>
+            <b>Note:</b> you may need to install the binstar client (<code>conda install
+            binstar</code>).
```
Comment (pzwang) [inline application/simple/templates/index.html:331]: Might be clearer as: "you may need to install the binstar client by running:"

File: application/simple/templates/index.html [hunk 4]
```diff
@@ -308,6 +308,39 @@
...
+          </p>
+
+          <p>
+            You can also create an environment with a <code>environment.yml</code>
+            hosted in anaconda.org as in:
```
Comment (pzwang) [inline application/simple/templates/index.html:336]: I think this sentence needs to be a little more clear.. I'm not quite sure what it means?

File: application/simple/templates/index.html [hunk 5]
```diff
@@ -308,6 +308,39 @@
...
+          </p>
+
+          {% call code.block("bash") %}
+$ conda env create malev/data-science
```
Comment (pzwang) [inline application/simple/templates/index.html:340]: Again, remove the `$` and also somehow explain that "malev/data-science" is a part that they will need to substitute in their own name/env.

**Comment by malev** [comment]
@pzwang this re will be deprecated soon. The doc about conda envs will be consolidated in main conda docs

**Pull request closed without merging**